docs/csharp/quick-starts/index.md

```diff
@@ -19,44 +20,52 @@ The first lessons explain C# concepts using small snippets of code. You'll
 learn the basics of C# syntax and how to work with data types like strings,
 numbers, and booleans. It's all interactive, and you'll be writing code
 within minutes. These first lessons assume no prior knowledge of
-programming or the C# language.
+programming or the C# language. 
+
+All the quick starts following the Hello World lesson are available using
+the online browser experience, or on your own development
+environment. At the end of each lesson, you decide if you want to continue
+with the next quick start online or on your own machine. There are links
+to help you setup your environment and continue with the next quick start
+on your machine.
 
 ## [Hello world](hello-world.yml)
 
-In the [Hello world](hello-world.yml) lesson, you'll create the most basic
+In the [Hello world](hello-world.yml) quick start, you'll create the most basic
 C#. You'll explore the `string` type and how to work with text.
 
 ## [Numbers in C#](numbers-in-csharp.yml)
 
-In the [Numbers in C#](numbers-in-csharp.yml) lesson, you'll learn
+In the [Numbers in C#](numbers-in-csharp.yml) quick start, you'll learn
 how computers store numbers and how to perform calculations with different
 number types. You'll learn the basics of rounding, and how to perform
-mathematical calculations using C#.
+mathematical calculations using C#. This quick start is also available
+[to run locally on your machine](numbers-in-csharp-local.md).
 
-This lesson assumes that you have finished the [Hello world](hello-world.yml) tutorial.
+This quick start assumes that you have finished the [Hello world](hello-world.yml) tutorial.
 
 ## [Branches and loops](branches-and-loops.yml)
 
 The [Branches and loops](branches-and-loops.yml) lesson teaches the basics of selecting
 different paths of code based on the values stored in variables. You'll learn the
 basics of control flow, which is the basics of how programs make decisions and choose
```

Comment:
basics of how programs --> basis of how programs

Reply:
nice suggestion!